src/core/shaderlib/picking/shadertools-picking.md

```diff
@@ -1,77 +0,0 @@
-# picking (Shader Module)
```

Comment:
Assume you made sure that any improvements to the text here was copied to luma.gl.

Reply:
yep, I double checked, I have ported all the text to `luma.gl/src/shadertools/modules/picking/README.md`.